pmlc/rt/level_zero/level_zero_probe.cc

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Intel Corporation
+
+#include <vector>
+
+#include "pmlc/rt/level_zero/level_zero_utils.h"
+#include "pmlc/util/logging.h"
+
+int main(int argc, char **argv) {
+  ze_result_t result = ZE_RESULT_NOT_READY;
+  try {
+    result = zeInit(0);
+  } catch (std::exception &e) {
+    IVLOG(1, "failed to init level zero driver, result:" << result << " info:"
+                                                         << e.what());
+    throw;
```

Comment:
Better to return a non-zero result here rather than throwing

Reply:
Done.